crt_portal/static/js/complaint_view_filters.js

```diff
@@ -292,6 +303,26 @@
     }
   };
 
+  /** Produces a new multi-field value given one of its constituent parts. */
+  function buildMultiValue(target, newValue) {
+    const fieldName = target
```

Comment:
This breaks with per page and grouping because those fields don't have ids

Reply:
I think I've addressed this; can you take another look? (If you're still seeing the issue, can give you me steps to reproduce?)